### ISSUE TITLE
Atom crashing after start

### ISSUE THREAD

**Issue by mz-gh** [yongkangchen/atom-ctags#103]
Hi!

After installing atom-ctags through apm install, Atom crashes after a few seconds.
Deleting Atom-ctags fixes the problem, but after reinstall the problem comes up again.

OS: Mageia 4.1
Atom: 0.207.0 (happened also with 0.206.0)
Atom-ctags version: 4.1.2

This is what I found in dmesg (3 consecutive starts):
[ 3986.288171] atom[4927]: segfault at 0 ip 00007f05887e3789 sp 00007f0589575b40 error 4 in ctags.node[7f05887e0000+a000]
[ 4343.543522] atom[6165]: segfault at 0 ip 00007fce8bc63789 sp 00007fce8c9f5b40 error 4 in ctags.node[7fce8bc60000+a000]
[ 4836.282177] atom[6628]: segfault at 0 ip 00007f0196c2b789 sp 00007f01979bdb40 error 4 in ctags.node[7f0196c28000+a000]

Cheers!

**Comment by yongkangchen** [comment]
Did you try install `atom-ctags` through atom setting view?

**Comment by mz-gh** [comment]
Yes, I also tried through Settings -> Install and the same issue happens.

**Comment by mz-gh** [comment]
I have played with it a little more and it seems that adding a directory with lots of data as project folder makes atom-ctags crash.
The directory I had was about 1.2GB big containing different sources (mostly java) and a few other resources (xmls, gzips, pdfs).

Maybe adding folders of such size is not very common, but a warning or error message would be helpful in such cases (with possibly disabling the package, as the user won't have a chance to remove the project folder in atom because it continuously crashes).

Cheers!

**Comment by kianw** [comment]
I have a similar situation (on OS X).  The repo is about a gigabyte, across 7300 files. Most of that is in a log directory, and in .git/objects

**Comment by yongkangchen** [comment]
According to #107, can you please try remove .tags file and try if atom start normal? BTW, please tell me the size of your .tags file.

**Comment by kianw** [comment]
Not, the OP, but removing the .tags file did allow the project to open. My .tags file was 15,333,771 bytes. I received this message when I opened the project after removing the .tags: "Stopped: Build more than 5 seconds, check if /Users/kian/Documents/repo/jiff3 contain too many file.
Suggest that add CmdArgs at atom-ctags package setting, example:
--exclude=some/path --exclude=some/other"

I think I'll add some exclude paths, notably ./log and ./.git

**Comment by abrookins** [comment]
Same problem, with a repo that has a directory containing about 1 GB worth of data. In my case, though, deleting the tags file and uninstalling/reinstalling didn't help -- I can't run atom on this project with atom-ctags enabled. That's a bummer because the package had been working pretty well!
